bin/Powheg/run_pwg_condor.py

```diff
@@ -77,9 +77,8 @@ def prepareCondorScript( tag, i, folderName, queue, SCALE = '0', njobs = 0, runI
    f.write('periodic_remove         = JobStatus == 5  \n')
    f.write('WhenToTransferOutput    = ON_EXIT_OR_EVICT \n')
    f.write('transfer_output_files   = "" \n')
-   if njobs > 0:
-       f.write('queue '+str(njobs)+'\n')
- 
+   #if njobs > 0:
+   #    f.write('queue '+str(njobs)+'\n')
```

Comment:
Hi, I don't know about this physics process but the queue argument should not be removed from the condor config.

Reply:
Hi, the reason I removed these lines is that they appear to cause a bug with the jobs not getting submitted to the batch using HTCondor, which is related to this commit: https://github.com/cms-sw/genproductions/commit/03ef366e8c7837aa0a6e93f8d7dc710d4ef029f1#diff-306a635579daba4fb10220a64e87ff0b29416fe9cfa2acf8bb8abd5391f8b7cd. 

The bug seems to happen because the "-queue' option is given twice when njobs>0 - once in the condor config (due to the lines you highlighted above), and once when submitting the jobs e.g this line: https://github.com/cms-sw/genproductions/blob/master/bin/Powheg/run_pwg_condor.py#L204

I think the bug should affect anyone trying to use the script with numJobs>0 but I wasn't sure how you wanted to deal with it in the end. So I just left these commented lines in the script. I can remove the comments if you like but then others will run into the same issue in the future. Perhaps there is another way you prefer to fix this - please let me know how you want me to proceed anyway

Reply:
Hi Daniel, ok, could you remove the occurrence in L204 then, please?
```
        runCommand ('condor_submit ' + condorfile + ' -queue '+ str(njobs))
 ```

We need the submit files to be complete so that we can use HTCondor DAGs to chain the multiple stages together in
https://github.com/cms-sw/genproductions/blob/master/bin/Powheg/run_pwg_parallel_condor.py

Reply:
Hi, sure I can do this but then I think I should also remove the "-queue" for every occurrence of condor_submit, even the ones that weren't directly causing my issues since they will have similar issues if someone runs with other options. Shall I do this?

Reply:
or rather only change the other occurrences where njobs is passed to prepareCondorScript, since the default value is 0 in which case the queue option doesn't get added

Reply:
Probably it's easiest to just give the number of jobs to the condor script `prepareCondorScript` everytime? I.e. just pass `njobs=1` when there was a `-queue 1` afterwards.

Reply:
OK I have modified the script as you suggested and uncommented those 2 lines. Please let me know if I need to do anything further